zipkin-thrift/src/main/thrift/com/twitter/zipkin/zipkinCore.thrift

```diff
@@ -204,5 +204,19 @@ struct Span {
   6: list<Annotation> annotations, # all annotations/events that occured, sorted by timestamp
   8: list<BinaryAnnotation> binary_annotations # any binary annotations
   9: optional bool debug = 0       # if true, we DEMAND that this span passes all samplers
+  /**
+   * Microseconds from epoch, usually taken from the first annotation. Ex.
+   * SERVER_RECV.timestamp
+   */
+  10: optional i64 start_ts,
```

Comment:
What is the benefit of capturing this explicitly instead of the usual cs/cr? It feels like it's going to complicate other code paths as they need to handle dual representation.